test/celeritas/geo/Geometry.test.cc

```diff
@@ -0,0 +1,211 @@
+           "absorber_lv_41", "gap_lv_42", "absorber_lv_42", "gap_lv_43",
+           "absorber_lv_43", "gap_lv_44", "absorber_lv_44", "gap_lv_45",
+           "absorber_lv_45", "gap_lv_46", "absorber_lv_46", "gap_lv_47",
+           "absorber_lv_47", "gap_lv_48", "absorber_lv_48", "gap_lv_49",
+           "absorber_lv_49"};
+    return make_span(vols);
+}
+
+auto TestEm3Test::reference_avg_path() const -> SpanConstReal
+{
+    static const real_type paths[]
+        = {6.228,  0.05519, 0.1539, 0.07731, 0.1966, 0.08251, 0.2072, 0.1047,
+           0.214,  0.09647, 0.2131, 0.0836,  0.1949, 0.07431, 0.1902, 0.08058,
+           0.2238, 0.1,     0.2521, 0.1161,  0.2957, 0.1093,  0.3308, 0.1205,
+           0.2553, 0.09231, 0.2359, 0.1077,  0.2468, 0.1002,  0.3126, 0.128,
+           0.3064, 0.124,   0.3066, 0.1245,  0.2827, 0.1083,  0.327,  0.1099,
+           0.2887, 0.132,   0.3088, 0.1347,  0.3605, 0.1282,  0.3057, 0.124,
+           0.2988, 0.1253,  0.3154, 0.1269,  0.3396, 0.1155,  0.3211, 0.1556,
+           0.262,  0.1006,  0.2587, 0.09687, 0.2327, 0.1018,  0.272,  0.09303,
+           0.2311, 0.09289, 0.366,  0.1,     0.2507, 0.1136,  0.3359, 0.116,
+           0.2615, 0.09728, 0.243,  0.09038, 0.2341, 0.09307, 0.2336, 0.0999,
+           0.2512, 0.1046,  0.2928, 0.1088,  0.2193, 0.09185, 0.2328, 0.08289,
+           0.1896, 0.07814, 0.1868, 0.0777,  0.2072, 0.09048, 0.1992, 0.08091,
+           0.1917, 0.07575, 0.1777, 0.07219, 0.1664};
+    return make_span(paths);
+}
+
+//---------------------------------------------------------------------------//
+
+class SimpleCmsTest : public HeuristicGeoTestBase
+{
+  protected:
+    const char* geometry_basename() const override { return "simple-cms"; }
+
+    HeuristicGeoScalars build_scalars() const final
+    {
+        HeuristicGeoScalars result;
+        result.lower = {-30, -30, -700};
+        result.upper = {30, 30, 700};
+        return result;
+    }
+
+    size_type     num_steps() const final { return 1024; }
+    SpanConstStr  reference_volumes() const final;
+    SpanConstReal reference_avg_path() const final;
+};
+
+auto SimpleCmsTest::reference_volumes() const -> SpanConstStr
+{
+    static const std::string vols[] = {"vacuum_tube",
+                                       "si_tracker",
+                                       "em_calorimeter",
+                                       "had_calorimeter",
+                                       "sc_solenoid",
+                                       "fe_muon_chambers",
+                                       "world"};
+    return make_span(vols);
+}
+
+auto SimpleCmsTest::reference_avg_path() const -> SpanConstReal
+{
+    static const real_type paths[]
+        = {32.85, 133.7, 59.85, 104.9, 80.73, 71.22, 40.84};
+    return make_span(paths);
+}
+
+//---------------------------------------------------------------------------//
+
+#if CELERITAS_USE_VECGEOM
+// For some reason, the geometry seems to be "outside" when initializing?
```

Comment:
Maybe related to the `PI` and `TWOPI` constants in the gdml? I switched the delta angles to degrees and it seems to work.

Reply:
Yay! That did allow it to initialize inside, but somehow not enough tracks are happening inside the innermost sphere:
```
i            ref_path           avg_path         Difference
0              0.1446  0.0404690400905626  -0.720131119705653
1               2.297    2.34027647482371  0.0188404330969566
2               3.332     3.3933447187659  0.0184107799417457
```

I'll open up a new issue to try to track this discrepancy down.